src/main/java/homework_4/custom_annotation/Info.java

```diff
@@ -0,0 +1,9 @@
+package homework_4.custom_annotation;
+
+import java.lang.annotation.*;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.FIELD)
+@Inherited
+public @interface Info {
```

Comment:
Opt: please add examples of usage if this annotation in Main method. I can't actually see how it works with console. As far as I understand it prints fields when printInfo is called, right?

Reply:
I've made a test for showing how it works. The last test in the ChairConstructorAnnotationTest. I use @Info for a new Class Cat, that doesn't have any relationship to the "chair classes". It allows to prints only marked fields.